cartManagement.html

```diff
@@ -116,14 +118,14 @@ <h3 class="card-heading">Nike Men's Flex 2020</h3>
             <div class="Card-horizontalImage-text">
               <img
                 class="card-img Card-horizontalImage-text-img Card-horizontalImage-text-img-Ecommerce cart_card_imgresize"
-                src="https://images.unsplash.com/photo-1491553895911-0055eca6402d?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=580&q=80"
+                src="https://pyxis.nymag.com/v1/imgs/4a7/29b/4ef2f518834dd81d8ceb374caa54fd5171-22-plant-gifts.rsquare.w700.jpg"
```

Comment:
You can add loading="lazy" to images. It will help in performance of the app.

Reply:
Yess I will do that